_data/versions.yml

```diff
@@ -1,8 +1,11 @@
 - name: "4.4"
-  flavor: "beta2"
-  release_date: "29 January 2025"
-  release_notes: "/article/dev-snapshot-godot-4-4-beta-2/"
+  flavor: "beta3"
+  release_date: "7 February 2025"
+  release_notes: "/article/dev-snapshot-godot-4-4-beta-3/"
   releases:
+    - flavor: "beta2"
```

Comment:
Oops this was a typo, should be `name` (no idea what it differs from the top level one). I'll fix it up.